tests/e2e.rs

```diff
@@ -402,5 +458,33 @@ fn client_server_integration(
         &format!("No queryable databases owned by {}", first_entity_0),
     )?;
 
+    update_profile(
```

Comment:
Assuming you agree with my feedback on only having to update a subset of fields, should we test
* Update field 1, field 2
* Profile has field 1 / field 2
* Update field 2, field 3
* Profile has field 1 / new field 2 / field 3?